code_generation/templates/src/power_grid_model/core/dataset_class_maps.py.jinja

```diff
@@ -10,12 +10,12 @@
 {%- set components = all_map['input'].keys() %}
 
 from enum import Enum, EnumMeta
-from typing import Any, Mapping
-
-# pylint: disable=invalid-name
+from typing import Any, Mapping, TypeAlias, TypeVar
```

Comment:
Why is this needed? What is Pylint complaining about?

Reply:
what do you mean? is this about the correct line?

Reply:
Oops. Added a comment on the correct line.